samba-backup/rootfs/scripts/main.sh

```diff
@@ -21,7 +21,7 @@ function create-backup {
     # do we need a partial backup?
     if [[ -n "$EXCLUDE_ADDONS" || -n "$EXCLUDE_FOLDERS" ]]; then
         # include all installed addons that are not listed to be excluded
-        addons=$(ha addons --raw-json | jq -rc '.data.addons[] | select (.installed == true) | .slug')
+        addons=$(ha addons --raw-json | jq -rc '.data.addons[] | .slug')
```

Comment:
```suggestion
addons=$(ha addons --raw-json | jq -rc '.data.addons[] | select (.installed != false) | .slug')
```
Per Thomas's comment [here](https://community.home-assistant.io/t/samba-backup-has-stopped-backing-up-my-installed-addons/435589/12?u=centralcommand) this might work better.